code/game/gamemodes/changeling/powers/revive.dm

```diff
@@ -52,15 +52,15 @@
 			IO.rejuvenate()
 			IO.trace_chemicals.Cut()
 		H.remove_all_embedded_objects()
-	user.updatehealth()
+	user.status_flags &= ~(FAKEDEATH)
+	user.updatehealth("revive sting")
```

Comment:
Would it be better and more accurate to say regenerative statis or regen statis instead of revive sting?

Reply:
I can do that. I was imitating the way line 55 in fakedeath.dm was written:
`user.update_stat("fakedeath sting")`
Should I change both?
It could be "regenerative stasis beginning" and "regenerative stasis ending"

Reply:
Hmmm, actually, that make sense for consistency sake. Lemme just try this PR out.